script/hassfest/manifest.py

```diff
@@ -23,7 +24,17 @@
 DOCUMENTATION_URL_PATH_PREFIX = "/integrations/"
 DOCUMENTATION_URL_EXCEPTIONS = {"https://www.home-assistant.io/hassio"}
 
-SUPPORTED_QUALITY_SCALES = ["gold", "internal", "platinum", "silver"]
+
+class QualityScale(IntEnum):
+    """Supported manifest quality scales."""
+
+    INTERNAL = -1
+    SILVER = 1
```

Comment:
I set INTERNAL to be negative because I don't think it fits in the regular quality scales.

And do we need to consider adding `NOT_SCORE = 0`? I think `None` is easier in this case but happy to adjust.